From 624f7ca87a521078ee0ed007bfdb3c4d2f0ff7fb Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 24 Nov 2023 10:04:56 +0100
Subject: [PATCH] unix: Use canonical mime-type for 3MF files

model/3mf is the canonical mime-type, but keep the
application/vnd.ms-3mfdocument alias.

See for details:
https://gitlab.freedesktop.org/xdg/shared-mime-info/-/merge_requests/179
---
 src/platform/unix/BambuStudio.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/platform/unix/BambuStudio.desktop b/src/platform/unix/BambuStudio.desktop
index ebd2df4a61da..8493077bf3da 100644
--- a/src/platform/unix/BambuStudio.desktop
+++ b/src/platform/unix/BambuStudio.desktop
@@ -5,7 +5,7 @@ Icon=BambuStudio
 Exec=bambu-studio %F
 Terminal=false
 Type=Application
-MimeType=model/stl;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;application/x-amf;
+MimeType=model/stl;model/3mf;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;application/x-amf;
 Categories=Graphics;3DGraphics;Engineering;
 Keywords=3D;Printing;Slicer;slice;3D;printer;convert;gcode;stl;obj;amf;SLA
 StartupNotify=false
-- 
2.43.0

